tests/search/bm25_feature/bm25_feature.rb

```diff
@@ -54,6 +105,33 @@ def assert_scores_for_query(query, exp_scores)
     end
   end
 
+  def use_sdfile(sdfile)
+    dest_sd = "#{dirs.tmpdir}test.sd"
+    command = "cp #{@test_dir}#{sdfile} #{dest_sd}"
+    success = system(command)
+    puts "use_sdfile(#{sdfile}): command='#{command}', success='#{success}'"
+    assert(success)
+    dest_sd
+  end
+
+  def postdeploy_wait(deploy_output)
+    wait_for_application(vespa.container.values.first, deploy_output)
+    wait_for_config_generation_proxy(get_generation(deploy_output))
+    wait_for_reconfig(600)
+  end
+
+  def redeploy(sdfile)
```

Comment:
Is it sufficient to use redeploy() from search_test.rb instead? If not, we should extend that instead of re-implementing this several places.